lib/ansible/modules/storage/hpe/hpe3par_cpg.py

```diff
@@ -0,0 +1,435 @@
+#!/usr/bin/python
+
+# (C) Copyright 2018 Hewlett Packard Enterprise Development LP
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.  Alternatively, at your
+# choice, you may also redistribute it and/or modify it under the terms
+# of the Apache License, version 2.0, available at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <https://www.gnu.org/licenses/>
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+author: "Hewlett Packard Enterprise (ecostor@groups.ext.hpe.com)"
+description:
+  - "Create and delete CPG on HPE 3PAR."
+module: hpe3par_cpg
+options:
+  cpg_name:
+    description:
+      - "Name of the CPG."
+    required: true
+  disk_type:
+    choices:
+      - FC
+      - NL
+      - SSD
+    description:
+      - "Specifies that physical disks must have the specified device type."
+    required: false
+  domain:
+    description:
+      - "Specifies the name of the domain in which the object will reside."
+    required: false
+  growth_increment:
+    default: -1.0
+    description:
+      - "Specifies the growth increment the amount of logical disk storage
+       created on each auto-grow operation.\n"
+    required: false
+  growth_increment_unit:
+    choices:
+      - MiB
+      - GiB
+      - TiB
+    default: GiB
+    description:
+      - "Unit of growth increment."
+    required: false
+  growth_limit:
+    default: -1.0
+    description:
+      - "Specifies that the autogrow operation is limited to the specified
+       storage amount that sets the growth limit.\n"
+    required: false
+  growth_limit_unit:
+    choices:
+      - MiB
+      - GiB
+      - TiB
+    default: GiB
+    description:
+      - "Unit of growth limit."
+    required: false
+  growth_warning:
+    default: -1.0
+    description:
+      - "Specifies that the threshold of used logical disk space when exceeded
+       results in a warning alert.\n"
```

Comment:
That newline is probably a conversion error ?

Reply:
Removed newline